app.rb

```diff
@@ -0,0 +1,122 @@
+require_relative 'book'
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
+require_relative 'classroom'
+
+class App
+  attr_accessor :books, :people, :rentals
+
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
```

Comment:
- [ ] Dear @maximoortelli, Good job adding the `app.rb` and `main.rb` files, but currently I could not get your app to run, I see the first and obvious issue with your `rental.rb` file having the error `undefined local variable or method 'person' for Rental:Class (NameError)` I address this but the app keeps throwing error one after the other. Please make sure the app is working fine and re-submit for a re-review. 

### The issue 

https://github.com/maximoortelli/OOP_School_Library/assets/69077061/4bfcd035-216c-4bcd-9555-5a4f1b54cca5

Reply:
Done, thank you for the review. Im fixed the bugs.